ObjectMemory_FreeList.c

```diff
@@ -3,7 +3,7 @@
 #include "ObjectMemory_Storage.h"
 #include "RealWordMemory.h"
 
-Word currentSegment = (0 % HeapSegmentCount);
+Word currentSegment = 0;
```

Comment:
I know @acf said this tripped up a compiler before, `clang` warns on it too and it's an unnecessary calculation.

Reply:
yeah DICE just crashed.  I'd forgotten about that as I moved onto VBCC :-)